### PR TITLE
Fix linux build

### PR DESCRIPTION
**Pull request by shenhavba**

### CODE REVIEW

File: sdk/file_sdk/UnitTests/xmp_ut/xmp_ut.pro [hunk 1]
```diff
@@ -22,13 +22,14 @@ win32:LIBS += -L$$REPO_ROOT/third_party/lib/xmp
...
-    LIBS +=  -lrmscryptod -lrmsd -lmodxmpfiled -lmodfilecommond -lXMPCoreStaticD -lXMPFilesStaticD
+    LIBS +=  -lrmscryptod -lrmsd -lmodxmpfiled -lmodfilecommond
+    win32:LIBS += -lXMPCoreStaticD -lXMPFilesStaticD
 } else {
     LIBS +=  -lrmscrypto -lrms -lmodxmpfile -lmodfilecommon
     win32:LIBS += -lXMPCoreStatic -lXMPFilesStatic
 }
 
-unix:!mac:LIBS += -lstaticXMPFiles -lstaticXMPCore
+unix:!mac:LIBS += -L/usr/lib/xmp -lXMPCore -lXMPFiles -ldl
```
Comment (assapir) [inline sdk/file_sdk/UnitTests/xmp_ut/xmp_ut.pro:32]: >unix:!mac:LIBS += -L/usr/lib/xmp [](start = 0, length = 32)

shouldn't be. use linux default (/usr/lib) and explicitly include <xmp/...> for simplicity

File: third_party/ReadMe.txt [hunk 2]
```diff
@@ -2,12 +2,13 @@ To compile file_sdk:
 
 1. Download Adobe xmp sdk from: http://download.macromedia.com/pub/developer/xmp/sdk/XMP-Toolkit-SDK-CC201607.zip
 2. Extract the zip file
-3. Follow the instructions in third-party folder and then follow the instruction in the build folder to generate static library.
+3. Follow the instructions in <xmp-sdk>/third-party folder and then follow the instruction in the <xmp-sdk>/build folder to generate static library.
 
 For Windows:
 4. Copy XMP-Toolkit-SDK-CC201607\public\include folder to rms-sdk-for-cpp\third_party\include\xmp
-5. Copy XMPCoreStatic.lib and XMPFilesStatic.lib to rms-sdk-for-cpp\third_party\lib\xmp
+5. Copy XMPCoreStatic.lib and XMPFilesStatic.lib to rms-sdk-for-cpp\third_party\lib\xmp (for debug build, append "D" to the name of the files: XMPFilesStaticD.lib, XMPFilesStaticD.lib)
```
Comment (assapir) [inline third_party/ReadMe.txt:9]: >(for debug build, [](start = 88, length = 17)

when building in debug

**Review by assapir — CHANGES_REQUESTED** [review]
:clock1:

File: sdk/file_sdk/UnitTests/xmp_ut/xmp_ut.pro (hunk 1 above)
Comment (shenhavba) [reply sdk/file_sdk/UnitTests/xmp_ut/xmp_ut.pro:32]: No, it shouldn't. its not include folder, its a lib folder.

**Review by assapir — APPROVED** [review]
:shipit:

File: third_party/ReadMe.txt (hunk 2 above)
Comment (shenhavba) [reply third_party/ReadMe.txt:9]: whats wrong with : "for debug build" ?

**Pull request merged**